File: POO/TP2/Practica Python/EJ8/Familia.py
class Familia:
    __personas:list
    __id_familia:0
    
    def __init__(self, id):
        self.__id_familia = id
        self.__personas = []
    
    def get_id(self):
        return self.__id_familia
    
    def set_id(self, id):
        self.__id_familia = id
        
    def get_personas(self):
        return self.__personas
    def add_persona(self, persona):
        self.__personas.append(persona)
    

File: POO/TP2/Practica Java/EJ10/Familia.java
package EJ10;

import java.util.ArrayList;

public class Familia {
    private Integer idFamilia = 0;
    private ArrayList<Persona> personas;

    public Familia(Integer idFamilia, ArrayList<Persona> personas) {
        this.idFamilia = idFamilia;
        this.personas = personas;
    }

    public Integer getIdFamilia() {
        return this.idFamilia;
    }

    public void setIdFamilia(Integer idFamilia) {
        this.idFamilia = idFamilia;
    }

    public ArrayList<Persona> getPersonas() {
        return personas;
    }

    public void setPersonas(ArrayList<Persona> personas) {
        this.personas = personas;
    }

    public void addpersona(Persona persona) {
        this.personas.add(persona);
    }
}
